src/lib/asn1/ASN1Reader.cpp

```diff
@@ -152,131 +142,110 @@ bool ASN1Reader::IsContained() const
 
 CHIP_ERROR ASN1Reader::GetInteger(int64_t & val)
 {
-    if (Value == nullptr)
-        return ASN1_ERROR_INVALID_STATE;
-    if (ValueLen < 1)
-        return ASN1_ERROR_INVALID_ENCODING;
-    if (ValueLen > sizeof(int64_t))
-        return ASN1_ERROR_VALUE_OVERFLOW;
```

Comment:
Why did this check go away?

Reply:
I'm not sure I follow...  This check is checking `ValueLen <= 8`.  Knowing that `ValueLen` is `uint32_t` does not tell us that....

Reply:
Ahh! Of course! Sorry for confusion. This check is returned now.